test/humanizer.js

```diff
@@ -99,7 +99,7 @@ describe('humanizer', function () {
     var h = humanizer({ round: true })
 
     assert.equal(h(3692131200000, { largest: 1 }), '117 years')
-    assert.equal(h(3692131200000, { largest: 2 }), '117 years')
+    assert.equal(h(3692131200000, { largest: 2 }), '116 years, 12 months')
```

Comment:
I'm not sure this test needs to change...I might be wrong, though. Why did you change it?